resources/views/components/banner.blade.php

```diff
@@ -10,8 +10,8 @@
 @endphp
 
 <style>
-    html {
-        margin-top: 50px;
+    aside, body {
```

Comment:
Interesting, why include `aside` here?

Reply:
The sidebar wasn't actually being pushed down with how it was before.